extensions.md

```diff
@@ -31,8 +31,17 @@ decided by those representations. Transport bindings (e.g.
 [AMPQ](amqp-transport-binding.md), [NATS](nats-transport-binding.md)) provide
 default placement for extensions, but an extension MAY require special
 representation when transported (e.g. tracing standards that require
-specific headers).
+specific headers). Extension authors SHOULD only require special
+representation in transport bindings where extensions integrate with
+pre-existing specs; extensions with custom transport bindings are much
+more likely to be dropped by middleware that does not understand the
+extension.
 
-## Documented Extensions
+As a convention, extensions of scalar types (`String`, `Binary`, `URI`,
```

Comment:
This PR added and Integer type. Should either list it here or add "e.g." to not list all of the scaler types.

Reply:
+1 to adding e.g.